x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_size_property.tsx

```diff
@@ -111,7 +111,11 @@ export class DynamicSizeProperty extends DynamicStyleProperty<SizeDynamicOptions
   getMbSizeExpression() {
     const rangeFieldMeta = this.getRangeFieldMeta();
     if (!this._isSizeDynamicConfigComplete() || !rangeFieldMeta) {
-      return null;
+      // return min of size to avoid flashing
+      // returning minimum allows "growing" of the symbols when the meta comes in
+      // A grow effect us less visually jarring as shrinking.
+      // especially relevant when displaying fine-grained grids using mvt
+      return this._options.minSize;
```

Comment:
Is minSize always provided? _isSizeDynamicConfigComplete fails if options.minSize does not exist. Maybe add a check for undefined?

Reply:
the typing in TS already enforces a valid number (https://github.com/elastic/kibana/blob/9134bc07d8e0ae02e729a81860cf9005f9deafad/x-pack/plugins/maps/common/descriptor_types/style_property_descriptor_types.ts#L161-L162).

I slightly rewrote the has-number check, and removed the use of `_` from this module.